### PR TITLE
Wootric Docs Design Update

### PR DESCRIPTION
**Pull request by victoriakoh**
# Task
- Improve Wootric docs design
- Add CSAT code snippet

### Landing page
screen shot 2018-09-11 at 13 19 28

### Docs page
screen shot 2018-09-11 at 17 43 43

### Docs navigation
screen shot 2018-09-11 at 17 43 48

### Docs page with search bar
screen shot 2018-09-11 at 17 43 59

### CSAT code snippet
screen shot 2018-09-11 at 17 40 53

### Language switcher
screen shot 2018-09-13 at 12 10 27

## Features
- Overall new docs design
- New docs landing page design
- New navigation
- New search bar
- Add CSAT code snippet

# How To Test
- Go to this branch
- Run `bundle exec middleman server`
- Launch http://localhost:4567

# Trello card: https://trello.com/c/XBlzvY8s

### CODE REVIEW

**Comment by prabhatjha** [comment]
@victoriakoh It looks great. The only problem I found was the collapse/expand of already expanded menu. For example, if I go to http://localhost:4567/api#declines , I would expect `declines` submenu to collapse when I click on `Declines` again. It only collapses when I click on other submenu.

File: source/includes/_api_index.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ language_tabs:
   - shell
 
 toc_footers:
-  - <a href='https://www.wootric.com/'>Sign Up</a>
-  - <a href='http://github.com/tripit/slate'>Documentation Powered by Slate</a>
+  - © 2018 Wootric&nbsp; | &nbsp;Docs powered by <a href='http://github.com/tripit/slate'>Slate</a>
```
Comment (marinhero) [inline source/includes/_api_index.md:8]: I wish markdown supported a way to keep the date current!

File: source/index.md [hunk 2]
```diff
@@ -1,18 +1,54 @@
...
+    <div class="wootric-doc-header">
+        <div class="wootric-doc-header__main">
+            Welcome to the Wootric documentation page!
+        </div>
+        <div class="wootric-doc-header__sub">
+            Select a documentation to get started
+        </div>
+    </div>
+    <div class="wootric-doc-landing-content">
+        <a class="wootric-doc-landing-content__type
```
Comment (marinhero) [inline source/index.md]: You are placing a block element inside an inline element. This is not ideal. Browsers won't tell you anything but keeping a valid markup gives you brownie points. Up to you if you want to change it or not (you can just set the `a` as `inline-block`)

File: source/index.md [hunk 3]
```diff
@@ -1,18 +1,54 @@
...
-* <a href="/ios" style="text-decoration:none;color:#000">Install Wootric using iOS SDK</a>
-<br><br>
-* <a href="/android" style="text-decoration:none;color:#000">Install Wootric using Android SDK</a>
-<br><br>
-* <a href="/api" style="text-decoration:none;color:#000">REST API</a>
-<br><br>
-* <a href="/webhooks" style="text-decoration:none;color:#000">Outgoing Webhooks</a>
-<br><br>
-</b>
+<div class="wootric-doc-landing">
```
Comment (marinhero) [inline source/index.md:1]: Pretty good BEM all across the file 👍

**Comment by hachibu** [comment]
Amazing updates!
- When the window gets small apply flex-direction: column.
- When I place my cursor on the edge of a front screen box it bounces infinitely.

File: source/layouts/layout.erb [hunk 4]
```diff
@@ -58,42 +58,85 @@ under the License.
...
+            <a href="/api" <% if current_page.url == '/api/' %>class="active"<% end %>>REST API</a>
+            <a href="/webhooks" <% if current_page.url == '/webhooks/' %>class="active"<% end %>>Outgoing Webhooks</a>
+          </div>
         </div>
-        <ul class="search-results"></ul>
       <% end %>
-      <div id="toc">
+      <div class="wootric-doc-topbar__space"></div>
+      <div class="wootric-doc-topbar__sign">
+        <a class="wootric-doc-topbar__sign--in" href="https://app.wootric.com/sign_in">sign in</a>
```
Comment (marinhero) [inline source/layouts/layout.erb]: `wootric-doc-topbar__sign-in` to keep your BEM 👌

File: source/layouts/layout.erb [hunk 5]
```diff
@@ -58,42 +58,85 @@ under the License.
...
+        <a class="wootric-doc-topbar__sign--up" href="https://app.wootric.com/sign_up">sign up free</a>
```
Comment (marinhero) [inline source/layouts/layout.erb]: `wootric-doc-topbar__sign--up` to keep your BEM 👌

**Comment by marinhero** [comment]
Excellent work! here are my observations:

- Ask Lisa if she wants to track the users that could sign_up from here so you add a query string parameter to the sign up URL to do so
- There are some weird blank spaces (see attached images)
- There are some navigation issues (see attached images)

screen shot 2018-09-12 at 11 15 02 am
screen shot 2018-09-12 at 11 13 45 am
screen shot 2018-09-12 at 11 18 32 am
screen shot 2018-09-12 at 11 23 08 am
screen shot 2018-09-12 at 11 21 22 am

**Comment by prabhatjha** [comment]
@marinhero The whitespace issue was there in the past too. My recommendation would be to deploy what Victoria has as the first iteration. Then we make subsequent changes. In my conversation with @victoriakoh we might be constrained by how slate works and is designed.

**Review by marinhero — APPROVED** [review]
It looks so much better now. :shipit: !

**Pull request merged**